routes/web.php

```diff
@@ -16,12 +18,25 @@
     ]);
 });
 
+// Dashboard route
 Route::get("/dashboard", fn() => Inertia::render("Dashboard"))->middleware(["auth", "verified"])->name("dashboard");
 
+// Course Materials routes
+Route::middleware(['web', 'auth'])->group(function (): void {
+    Route::resource('course-materials', CourseMaterialController::class);
+});
+
+// Profile routes
 Route::middleware("auth")->group(function (): void {
     Route::get("/profile", [ProfileController::class, "edit"])->name("profile.edit");
     Route::patch("/profile", [ProfileController::class, "update"])->name("profile.update");
     Route::delete("/profile", [ProfileController::class, "destroy"])->name("profile.destroy");
 });
 
+// Debug lessons route
```

Comment:
Please remove this route - as I understand it's for debugging, so it should be pushed